ga4gh/backend.py

```diff
@@ -571,6 +573,78 @@ def callSetsGenerator(self, request):
                 return self._noObjectGenerator()
             return self._singleObjectGenerator(callSet)
 
+    def rnaQuantificationGenerator(self, request):
+        """
+        Returns a generator over the (rnaQuantification, nextPageToken) pairs
+        defined by the specified request.
+        """
+        if request.rnaQuantificationId is None:
+            allRnaQuants = []
+            for dataset in self.getDataRepository().getDatasets():
+                for rnaQuant in dataset.getRnaQuantifications():
+                    allRnaQuants.append(rnaQuant)
+            return self._objectListGenerator(request, allRnaQuants)
+        else:
+            compoundId = datamodel.RnaQuantificationCompoundId.parse(
+                request.rnaQuantificationId)
+            dataset = self.getDataRepository().getDataset(compoundId.datasetId)
+            try:
+                rnaQuant = dataset.getRnaQuantification(
+                    compoundId.rnaQuantificationId)
+            except exceptions.RnaQuantificationNotFoundException:
+                return self._noObjectGenerator()
+            return self._singleObjectGenerator(rnaQuant)
+
+    def expressionLevelGenerator(self, request):
+        """
+        Returns a generator over the (expressionLevel, nextPageToken) pairs
+        defined by the specified request.
+
+        Currently only supports searching over a specified rnaQuantification
+        """
+        rnaQuantificationId = request.rnaQuantificationId
+        compoundId = datamodel.RnaQuantificationCompoundId.parse(
+            request.rnaQuantificationId)
+        dataset = self.getDataRepository().getDataset(compoundId.datasetId)
+        rnaQuant = dataset.getRnaQuantification(rnaQuantificationId)
+        expressionLevelId = request.expressionLevelId
+        featureGroupId = request.featureGroupId
```

Comment:
See my other notes about `FeatureGroup`. I want to make sure you're not relying on entities and mechanisms that aren't there in Sequence Annotations.